composer.json

```diff
@@ -5,25 +5,27 @@
 	"homepage": "https://github.com/studiopress/genesis-simple-menus",
 	"license": "GPL-2.0-or-later",
 	"require": {
-		"php": "^5.3 || ^7",
+		"php": "^5.6 || ^7",
 		"composer/installers": "^1"
 	},
 	"require-dev": {
 		"php": "^5.6 || ^7",
-        "dealerdirect/phpcodesniffer-composer-installer": "*",
-        "squizlabs/php_codesniffer": "^3.3.1",
-        "phpcompatibility/phpcompatibility-wp": "*",
-		"wp-coding-standards/wpcs": "^1"
+      "dealerdirect/phpcodesniffer-composer-installer": "*",
+      "squizlabs/php_codesniffer": "^3.3.1",
+      "phpcompatibility/phpcompatibility-wp": "*",
+		"wp-coding-standards/wpcs": "^1",
+		"wp-cli/i18n-command": "^2.1"
 	},
 	"config": {
 		"sort-order": true
 	},
 	"minimum-stability": "dev",
 	"prefer-stable": true,
 	"scripts": {
-        "phpcs": "phpcs --standard=WordPress --ignore=vendor/,node_modules/,assets/ --extensions=php -p ./",
-        "phpcs-compat": "phpcs --extensions=php --standard=PHPCompatibilityWP --ignore=vendor/,node_modules/,assets/ --runtime-set testVersion 5.6- -p ./",
-        "phpcbf": "phpcbf --standard=WordPress --ignore=vendor/,node_modules/,assets/ --extensions=php -p ./"
+    "phpcs": "phpcs --standard=WordPress --ignore=vendor/,node_modules/,assets/ --extensions=php -p ./",
+    "phpcs-compat": "phpcs --extensions=php --standard=PHPCompatibilityWP --ignore=vendor/,node_modules/,assets/ --runtime-set testVersion 5.6- -p ./",
+    "phpcbf": "phpcbf --standard=WordPress --ignore=vendor/,node_modules/,assets/ --extensions=php -p ./",
+    "makepot": "wp i18n make-pot . languages/genesis-simple-menus.pot --headers='{ \"Report-Msgid-Bugs-To\": \"StudioPress <translations@studiopress.com>\" }' --exclude=vendor/ --quiet"
 	},
 	"support": {
 		"issues": "https://github.com/studiopress/genesis-simple-menus/issues",
```

Comment:
Hm, looks like there's spacing issues throughout the file. Spaces instead of tabs.

Reply:
Do we have any convention about this? The other problem was that I did use tabs instead of space, and now, the whole file is using spaces. The main difference, I guess, is that tabs are using 2 spaces instead of 4.

Reply:
In our repos, we prefer the indent style to be tabs, just for consistency sake.

Reply:
Sounds good to me. I'll update that file.

Reply:
I've already fixed this in the current branch, but github is not showing the newest commit here 🤷‍♂

Reply:
Yeah, looks like your instincts were right, @marksabbath ... guess we need to close this PR and open a new one on the same branch and see if that fixes it.